django/contrib/admindocs/views.py

```diff
@@ -254,11 +255,19 @@ def get_context_data(self, **kwargs):
                 verbose = func.__doc__
                 if verbose:
                     verbose = utils.parse_rst(utils.trim_docstring(verbose), 'model', _('model:') + opts.model_name)
-                fields.append({
```

Comment:
I think it would be nice to have no output if there's no description instead of "None" I think something like `verbose = verbose or ''` would accomplish this.